### PR TITLE
Added support for Android 6.0 runtime permissions.

### PR DESCRIPTION
**Pull request by alejandrohall**

### CODE REVIEW

File: library/src/main/java/org/firezenk/audiowaves/Visualizer.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import android.graphics.drawable.GradientDrawable;
 import android.os.Handler;
 import android.util.AttributeSet;
+import android.util.Log;
```
Comment (FireZenk) [inline library/src/main/java/org/firezenk/audiowaves/Visualizer.java:9]: Why?

**Comment by alejandrohall** [comment]
I was debugging and i forgot deleite it.

File: sample/src/main/java/org/firezenk/audiowaves/sample/AudioWavesApplication.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import android.app.Application;
+
+import com.karumi.dexter.Dexter;
+
+/**
+ * Created by alejandrohall on 13/03/16.
+ */
+public class AudioWavesApplication extends Application {
+
+
```
Comment (jzeferino) [inline sample/src/main/java/org/firezenk/audiowaves/sample/AudioWavesApplication.java:12]: Unnecessary empty line.

**Review by jzeferino — APPROVED** [review]

**Comment by xi23da** [comment]
您好，您发的邮件已经收到，请放心

**Pull request closed without merging**